app/components/market/application-action/choose-action.html

```diff
@@ -4,6 +4,7 @@
         <h1>
             <img class="logo" [src]="application.icon" height="30" width="30" alt="logo" *ngIf="application?.icon">
             <span>Action selection for <span [title]="application.description">{{application?.name}}</span></span>
+            <span>(<a (click)="openLink(application?.readme)" href="javascript:void(0)">view readme</a>)</span>
```

Comment:
Should we wrap this in an 'if' block in case the readme doesn't exist?

Reply:
Like the Icon file we just "assume" the readme file is there. If this becomes an issue, or we end up needing more than one readme (per folder/action), then we can look at a more encompassing solution by adding the icon and readme settings into the index.json file.